profile/README.md

```diff
@@ -4,4 +4,4 @@ The Open Quantum Safe (OQS) project is an open-source project that aims to suppo
 
 OQS consists of two main lines of work: [liboqs](https://github.com/open-quantum-safe/liboqs), an open source C library for quantum-resistant cryptographic algorithms, and prototype integrations into [protocols and applications](https://openquantumsafe.org/applications/), including a [provider for the widely used OpenSSL library](https://github.com/open-quantum-safe/oqs-provider).  These tools support [research](https://openquantumsafe.org/research/) by ourselves and others.
 
-All of our development takes place on our [GitHub](https://github.com/open-quantum-safe) repositories.  We welcome new contributors interested in joining our team.  Feel free to begin participating on GitHub or say hello on our [Discord server](https://discord.gg/qRfMantKwc) on the [#oqs-general channel](https://discord.com/channels/1202723482224295936/1203395992003678238).  We are grateful to the organizations that have supported and continue to support our work, and the academic, industry, public sector, and individual contributors who participate in the project.
+All of our development takes place on our [GitHub](https://github.com/open-quantum-safe) repositories.  We welcome new contributors interested in joining our team.  Feel free to begin participating on GitHub or say hello on our [Discord server](https://discord.gg/qRfMantKwc) on the [#oqs-general channel](https://discord.com/channels/1202723482224295936/1203395992003678238). Be aware that Discord shall not be used for technical discussions such as to ascertain all relevant technical background is available openly and in full in github.  We are grateful to the organizations that have supported and continue to support our work, and the academic, industry, public sector, and individual contributors who participate in the project.
```

Comment:
I would make this sound a little less intimidating :)

```suggestion
All of our development takes place on our [GitHub](https://github.com/open-quantum-safe) repositories.  We welcome new contributors interested in joining our team.  Feel free to begin participating on GitHub or say hello on our [Discord server](https://discord.gg/qRfMantKwc) on the [#oqs-general channel](https://discord.com/channels/1202723482224295936/1203395992003678238). If you'd like to share a bug report, ask a usage-related question, or start some other technical discussion, we ask that you open a GitHub [discussion](https://github.com/orgs/open-quantum-safe/discussions) or file an issue on an appropriate [repository](https://github.com/orgs/open-quantum-safe/repositories) rather than posting in Discord. This ensures that all technical background about OQS is archived in one place, so that others can benefit from it in the future. You will also likely receive a quicker response this way, as our dev team does not actively triage technical queries from Discord.

We are grateful to the organizations that have supported and continue to support our work, and the academic, industry, public sector, and individual contributors who participate in the project.
```